docs/application/tizen-studio/native-tools/ui-builder/custom-ui-component.md

```diff
@@ -45,7 +45,7 @@ A custom component requires an EDJ resource file (`[ComponentName].edj`) and an
     |-----------|-----------|-------------|----------------------------------------|-------|--------|------------------------------------|
     | -           | <component> | name          | Component  identifier, shown on the Palette, Outline view, and Properties view. This attribute must be identical to the XML file  name. | String  | Yes      | -                                    |
```

Comment:
The elements inside of \<, \> are considered as tags, so removed on github page. On the github page we can see empty spaces instead of `<component>`.

Reply:
done